packages/nx-plugin/src/generators/add-to-project/generator.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Tree,
+  formatFiles,
+  generateFiles,
+  readProjectConfiguration,
+  updateProjectConfiguration,
+} from '@nx/devkit';
+import * as path from 'path';
+import { join } from 'path';
+import { AddToProjectGeneratorSchema } from './schema';
+
+export async function addToProjectGenerator(
+  tree: Tree,
+  options: AddToProjectGeneratorSchema,
+) {
+  const projectConfiguration = readProjectConfiguration(tree, options.project);
+
+  const { root } = projectConfiguration;
+
+  if (tree.exists(path.join(root, 'code-pushup.config.json'))) {
+    console.log('Code PushUp already configured for this project');
+    return;
+  }
+
+  generateFiles(tree, path.join(__dirname, 'files'), root, options);
+
+  updateProjectConfiguration(tree, options.project, {
+    ...projectConfiguration,
+    targets: {
+      ...projectConfiguration.targets,
+      collect: {
+        executor: 'nx:run-commands',
```

Comment:
This will turn into a custom executor in a future PR i think?
Good, so it works ATM...

Reply:
Yes! But let's make the executor a separate issue.

Reply:
There is the issue about executor #63